leaderboard/wasmcloud.toml

```
@@ -0,0 +1,7 @@
+name = "leaderboard"
```

```suggestion
name = "Leaderboard"
```

This is the friendly name so we can capitalize 😄